lambdas/service/app.py

```diff
@@ -964,6 +966,15 @@ def list_catalogs():
     return app.catalog_controller.list_catalogs()
 
 
+def accessible_source_ids(catalog: CatalogName,
+                          request: AzulRequest
+                          ) -> Set[str]:
+    return {
+        source['sourceId']
+        for source in app.repository_controller.list_sources(catalog, request)
```

Comment:
if `list_sources` returns an empty list then downstream the implicit filter that's added to the ESquery is 
`{"sourceId" : {"is": []}`, which seems `ok`...

When we request the *repository* to list its sources for an authenticated user, does it include publicly accessible snapshots/sources within the response? i think i heard something last week about `making public snapshots available to everyone, even if their are not registered with Terra`.

Reply:
This might be a testcase to add for the IT PR, if the public snapshots are being returned within the list of sources/snapshots from TDR.

Reply:
> When we request the repository to list its sources for an authenticated user, does it include publicly accessible snapshots/sources within the response?

I believe the TDR team recently made most of the snapshots returned by the public service account accessible to all registered users. There appear to be some gaps but I think they correspond to old snapshots we're not going to be indexing.